Tribler/Core/defaults.py

```diff
@@ -163,6 +164,25 @@
 sessdefaults['http_api']['enabled'] = False
 sessdefaults['http_api']['port'] = -1
 
+# Credit mining config
+sessdefaults['credit_mining'] = OrderedDict()
+sessdefaults['credit_mining']['enabled'] = True
+sessdefaults['credit_mining']['max_torrents_per_source'] = 20
+sessdefaults['credit_mining']['max_torrents_active'] = 50
+sessdefaults['credit_mining']['source_interval'] = 100
+sessdefaults['credit_mining']['swarm_interval'] = 100
+sessdefaults['credit_mining']['share_mode_target'] = 3
+sessdefaults['credit_mining']['tracker_interval'] = 200
+sessdefaults['credit_mining']['logging_interval'] = 60
+sessdefaults['credit_mining']['boosting_sources'] = ["http://bt.etree.org/rss/bt_etree_org.rdf",
+                                                     "9e1a3fac737543d36c83d54818ed77620932ff80",
```

Comment:
That's a dispersy channel id. By default we want to automatically boost legal-predetermined channel.